crates/iceberg/src/spec/manifest.rs

```diff
@@ -924,7 +924,7 @@ impl TryFrom<i32> for ManifestStatus {
 }
 
 /// Data file carries data file path, partition tuple, metrics, …
-#[derive(Debug, PartialEq, Clone, Eq, TypedBuilder)]
+#[derive(Debug, PartialEq, Clone, Eq, Builder)]
```

Comment:
Why this change? `TypedBuilder` checks it in compile time, while `Builder` checks result in runtime.

Reply:
Oh, I see, thanks!